.github/workflows/elixir.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     - name: Set up Elixir
       uses: erlef/setup-beam@988e02bfe678367a02564f65ca2e37726dc0268f
       with:
-        elixir-version: '1.12.3' # Define the elixir version [required]
+        elixir-version: '1.5' # Define the elixir version [required]
```

Comment:
❓ What this meant to be 1.15? I think the latest is 1.16.

Reply:
Ok, I should change that

Reply:
See #288